client/profile-wizard/steps/business-details/index.js

```diff
@@ -45,22 +33,18 @@ export const BusinessDetailsStep = ( props ) => {
 		);
 	}
 
-	if ( selectiveBundleInstallSegmentation ) {
-		const initialValues = {
-			other_platform: profileItems.other_platform || '',
-			other_platform_name: profileItems.other_platform_name || '',
-			product_count: profileItems.product_count || '',
-			selling_venues: profileItems.selling_venues || '',
-			revenue: profileItems.revenue || '',
-		};
-
-		return (
-			<SelectiveFeaturesBusinessStep
-				{ ...props }
-				initialValues={ initialValues }
-			/>
-		);
-	}
-
-	return <BusinessDetailsStepWithExtensionList { ...props } />;
+	const initialValues = {
+		other_platform: profileItems.other_platform || '',
+		other_platform_name: profileItems.other_platform_name || '',
+		product_count: profileItems.product_count || '',
+		selling_venues: profileItems.selling_venues || '',
+		revenue: profileItems.revenue || '',
+	};
+
+	return (
+		<SelectiveFeaturesBusinessStep
```

Comment:
Its a nitpick but this could be renamed now. in the original implementation I had trouble naming it because both steps needed to be identifiable from each other.

Reply:
Tbh I can't come up with a good name for this either right now. Since the component renders the whole step, `BusinessDetailsStep` sounds accurate but that's going to be redundant with `business-details/index.js`. Merging the whole component with its parent sounds like overkill too.

Any suggestions?

Reply:
@ilyasfoo Maybe we could call it `FeaturesList` or `BusinessFeaturesList`

Reply:
`BusinessFeaturesList` it is!